zzzz_modular_occulus/code/modules/sprite_accessories/_accessory_hair.dm

```diff
@@ -131,4 +131,26 @@
 /datum/sprite_accessory/hair/teshari_fluffymohawk
 	name = "Teshari Fluffy Mohawk"
 	icon = 'zzzz_modular_occulus/icons/mob/jamini-hair.dmi'
-	icon_state = "teshari_fluffymohawk_s"
\ No newline at end of file
+	icon_state = "teshari_fluffymohawk_s"
+	
+///////// Dragonfruits more hair PR
+
+/datum/sprite_accessory/hair/halfshave
+	name = "Half-shaved"
+	icon = 'zzzz_modular_occulus/icons/mob/more_hair.dmi'
+	icon_state = "halfshave"
+	
+/datum/sprite_accessory/hair/halfshave
```

Comment:
This typepath is a duplicate of the first, it would just override the first one instead of creating a new entry.